kernel/h1/src/hil/flash/hardware.rs

```diff
@@ -28,14 +39,15 @@ pub trait Hardware {
     fn read_error(&self) -> u16;
 
     /// Set flash transaction parameters (word offset and size). The word offset
-    /// is relative to the start of flash and the size is one less than the
+    /// is relative to the start of a flash bank and the size is one less than the
```

Comment:
This `size-1` thing feels like a questionable choice to me.  It seems likely to lead to confusion in the future.

Why?

Reply:
The `Hardware` trait is really just a super thin layer above the actual HW registers. And the `-1` is just how it is in the actual hardware.

Reply:
Sigh...OK, can't fix that.